src/js/components/Navigation/HeaderBackTo.jsx

```diff
@@ -1,325 +1,325 @@
-            backToLink={backToLink}
-            backToLinkText={backToLinkText}
-            className="HeaderBackTo"
-            id="backToLinkTabHeaderBackTo"
-          />
-
-          <TopRowOneRightContainer
-            className="u-cursor--pointer"
-            style={{ paddingLeft: `${isCordova() ? '0 !important' : ''}` }}
-          >
-            {voterIsSignedIn && (
-              <Suspense fallback={<></>}>
-                <HeaderNotificationMenu />
-              </Suspense>
-            )}
-            {voterIsSignedIn ? (
-              <span onClick={this.goToSettings}>
-                {voterPhotoUrlMedium ? (
-                  <span>
-                    <div
-                      id="profileAvatarHeaderBar"
-                    >
-                      <LazyImage
-                        isAvatar
-                        src={voterPhotoUrlMedium}
-                        placeholder={avatarGeneric()}
-                        height={34}
-                        width={34}
-                        alt="Your Settings"
-                      />
-                    </div>
-                  </span>
-                ) : (
-                  <span>
-                    <IconButton
-                      classes={{ root: classes.iconButtonRoot }}
-                      id="profileAvatarHeaderBar"
-                      size="large"
-                    >
-                      <AccountCircle />
-                    </IconButton>
-                  </span>
-                )}
-              </span>
-            ) : (
-              <SignInButton toggleSignInModal={this.toggleSignInModal} />
-            )}
-          </TopRowOneRightContainer>
-        </Toolbar>
-        {showSignInModal && (
-          <Suspense fallback={<></>}>
-            <SignInModal
-              signInTitle="Sign In or Join"
-              signInSubTitle=""
-              toggleOnClose={this.closeSignInModal}
-              uponSuccessfulSignIn={this.closeSignInModal}
-            />
-          </Suspense>
-        )}
-      </AppBarForBackTo>
-    );
-  }
-}
-HeaderBackTo.propTypes = {
-  backToLink: PropTypes.string,
-  backToLinkText: PropTypes.string,
-  classes: PropTypes.object,
-  voterWeVoteId: PropTypes.string,
-};
-
-const styles = (theme) => ({
-  headerButtonRoot: {
-    paddingTop: 2,
-    paddingBottom: 2,
-    '&:hover': {
-      backgroundColor: 'transparent',
-    },
-    color: 'rgb(6, 95, 212)',
-    marginLeft: '1rem',
-    outline: 'none !important',
-    [theme.breakpoints.down('md')]: {
-      marginLeft: '.1rem',
-    },
-  },
-  iconButtonRoot: {
-    color: 'rgba(17, 17, 17, .4)',
-    outline: 'none !important',
-    paddingRight: 0,
-    '&:hover': {
-      backgroundColor: 'transparent',
-    },
-  },
-  stackedReturnAndShare: {
-    flexDirection: 'column',
-    alignItems: 'flex-start',
-  },
-});
-
-export default withStyles(styles)(HeaderBackTo);
-
+import { AccountCircle } from '@mui/icons-material';

```

Comment:
Hi Almee, could you please remove this file from your pull request? I don't see any changes, but Github is reacting as if you deleted this file and then added it beck in again. I would recommend spending some time researching how to fix this by searching Google. It might be a little bit complicated, but is worth it to learn.